services/core-web/src/components/mine/Permit/MinePermitTable.js

```diff
@@ -416,6 +422,19 @@ const childColumns = [
     align: "right",
     render: (text, record) => (
       <div>
+        <AuthorizationWrapper permission={Permission.ADMIN}>
+          <Popconfirm
+            placement="topLeft"
+            title={`Are you sure you want to Issue this permit as a Verifiable Credential to OrgBook ${record.email}?`}
```

Comment:
I don't see 'email' defined anywhere in the record - Is this working correctly?

Reply:
no it's not. i'll also tweak those auth wrappers.

Reply:
disregard my latest comment then haha

Reply:
I don't see any 'intesting' clause, so no tweak

Reply:
do you see any with a capital t, "inTesting"?